lib/Minz/Translate.php

```diff
@@ -231,10 +231,9 @@ public static function t(string $key, ...$args): string {
 			}
 		}
 
-		if (is_array($translation_value)) {
-			if (isset($translation_value['_'])) {
-				$translation_value = $translation_value['_'];
-			} else {
+		if (!is_string($translation_value)) {
+			$translation_value = is_array($translation_value) ? ($translation_value['_'] ?? null) : null;
+			if (!is_string($translation_value)) {
```

Comment:
```suggestion
		if (is_string($translation_value) === false) {
			$translation_value = is_array($translation_value) ? ($translation_value['_'] ?? null) : null;
			if (is_string($translation_value) === false) {
```
easier to read

Reply:
No, sorry. Not when dealing with pure booleans.